ggshield/cmd/hmsl/quota.py

```diff
@@ -20,7 +20,7 @@ def quota_cmd(
     **kwargs: Any,
 ) -> int:
     """
-    Get the number of remaining credits for today.
+    Get the remaining quotas for today.
```

Comment:
I don't know if "credits" was the correct word, but I believe "quotas" is not the correct one here. A quota is a limit on something. The only reason it would be plural if is you have several different limits (a soft-limit and a hard-limit for example).

Maybe "API calls" is what we want here, just like in `ggshield quota`?

Reply:
It may be more complex but what do you guys think of "consumption units" ?

Reply:
Sounds a bit complex indeed. I thought it would make sense to sync this with the dashboard, but I can't see any mention to this there. Maybe I haven't been looking in the right place?